lib/kube/proxy/portforward_spdy.go

```diff
@@ -199,7 +200,9 @@ func (h *portForwardProxy) forwardStreamPair(p *httpStreamPair, remotePort int64
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		if err := utils.ProxyConn(h.context, p.errorStream, targetErrorStream); err != nil {
+		// Use a unidirectional copy from Kube API to client.
+		// Enables error propagation from Kube API server to kubectl client.
```

Comment:
```suggestion
		_ = targetErrorStream.Close()
		// Enables error propagation from Kube API server to kubectl client.
```

I would keep the _ = targetErrorStream.Close() just like kubectl does to ensure the same behavior